src/localstorage.js

```diff
@@ -0,0 +1,40 @@
+// get the data from local storage
+export default function getDataLS() {
```

Comment:
Use classes to streamline your functions.

Reply:
Thank you